app/controllers/trips_controller.rb

```diff
@@ -0,0 +1,30 @@
+class TripsController < ApplicationController
+  def index
+    @trips = Trip.all
+  end
+
+  def new
+    @trip = Trip.new
+  end
+
+  def create
+    @trip = Trip.create(trip_params)
+
+    respond_with @trip, location: -> { trips_path }
+  end
+
+  private
+
+  def trip_params
+    params.require(:trip).permit(
+      :n_girls,
+      :n_boys,
+      :n_teachers,
+      :n_bus_drivers,
```

Comment:
Reconsider better, more descriptive attributes.
Also, if you are permitting all the attributes, consider using this instead:
```
params.require(:trip).permit!
```

Reply:
- Changed the attributes to be total_girls, total_boys, .. in sha `900b7c8`
- Added the bang for create with fixup `04819c8`